app/interfaces/MatchTeamStatsDTO.java

```diff
@@ -0,0 +1,167 @@
+package interfaces;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TeamStatsDTO contains all of the information necessary for the stats array list
+ *
+ * @author jshyo
+ *
+ */
+public class MatchTeamStatsDTO extends MatchTeamDTO{
+    // formatter
+    public static final DecimalFormat DF = new DecimalFormat( "#.##" );
+
+    // Display stats
+    private String opponent;
+    private String result;
+    private Integer pointsTotal;
+    private Integer pointsAgainst;
+    private List<TeamTossupValuesDTO> tossupValues;
+    private Integer TUH;
+    private Float PPTH;
+    private Float pnRatio;
+    private Float gnRatio;
+    private Integer bonusHeard;
+    private Integer pointsBonus;
+    private Integer pointsBounceback;
+    private Float PPB;
+
+    public MatchTeamStatsDTO(MatchTeamDTO matchTeamDTO, MatchDTO matchDTO){
+        // super constructor
+        super.setTeamId(matchTeamDTO.getTeamId());
+        super.setScore(matchTeamDTO.getScore());
+        super.setBouncebackPoints(matchTeamDTO.getBouncebackPoints());
+        super.setOvertimeTossups(matchTeamDTO.getOvertimeTossups());
+        super.setPlayers(matchTeamDTO.getPlayers());
+        super.setMatchId(matchTeamDTO.getMatchId());
+
+        // initialize tossupValues
+        tossupValues = new ArrayList<>();
+
+        // initialize opponent info
+        List<MatchTeamDTO> teams = matchDTO.getTeams();

```

Comment:
Can we move this out to a private method? It'll be cleaner than doing all the code inside the constructor